apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/edit/components/ImageSurveyBg.tsx

```diff
@@ -0,0 +1,25 @@
+import FileInput from "@formbricks/ui/FileInput";
+import { TSurvey } from "@formbricks/types/surveys";
+
+interface ImageSurveyBgBgProps {
+  localSurvey?: TSurvey;
+  handleBgChange: (url: string, bgType: string) => void;
+}
+
+export default function ImageSurveyBg({ localSurvey, handleBgChange }: ImageSurveyBgBgProps) {
+  return (
+    <div className="mb-2 mt-4 w-full rounded-lg border bg-slate-50 p-4">
+      <div className="mt-3 flex w-full items-center justify-center">
+        <FileInput
+          id="choices-file-input"
+          allowedFileExtensions={["png", "jpeg", "jpg"]}
+          environmentId={localSurvey?.environmentId}
+          onFileUpload={(url: string[]) => {
+            handleBgChange(url[0], "image");
+          }}
+          // fileUrl={localSurvey?.surveyBackground?.bg}
```

Comment:
Can you please remove this comment, if this is not required?

Reply:
Its required but I have commented it out for now as @pandeymangg will probably have a look at the file upload part over here as the loader doesnt stop even if the image is uploaded...
For reference - https://www.loom.com/share/7d9d6b63759447e5ae2f7ff006dc8f81?sid=1c790a83-8d23-49f4-b88e-2e9d559ba758